### DISCUSSION TITLE
Input data

### DISCUSSION THREAD

**Discussion by hadismasihi** [gempy-project/gempy#638] [Q&A]
Hi
I want to do subsurface modeling with the help of gempy borehole data. Is it possible?
And what is the required input data?
The images below show some example data from your site. What does this type of data each represent?
![AlesModel](https://user-images.githubusercontent.com/63184005/132938077-bec230c9-d724-4944-947c-f8576ac7c67f.JPG)
![contributor](https://user-images.githubusercontent.com/63184005/132938079-b121a3f3-4dec-4ca4-854a-f0682ad24fc9.JPG)

**Comment by AlexanderJuestel** [comment]
Hey @hadismasihi, 

Gemyp works with interface points. These points can be on the surface (such as for geological maps) or in the subsurface, such as boreholes. The image you displayed show the coordinates of your interface points and orientations, their rescaled values, the formation and series they belong to, the dip, azimuth and polarity for orientations and a smoothness factor. 

Let me know if we can help with anything else :)
